spec/application/application.js

```diff
@@ -0,0 +1,32 @@
+const {
+  messageId,
+  cloakedMessageId,
+  feedId,
+  tangle
+} = require('ssb-schema-definitions')()
+
+module.exports = {
+  type: 'tribe/application',
+  tangle: 'application',
+
+  staticProps: {
+    groupdId: {
+      ...cloakedMessageId,
+      required: true
+    },
+    version: {
+      type: 'string',
+      pattern: '^v1$',
+      required: true
+    }
+  },
+
+  props: {
+  },
+  addMember: {
+    type: 'object',
+    patternProperties: {
+      [msgIdRegex]: { type: 'integer' }
+    }
+  }
+}
```

Comment:
This file shouldn't be in this pr.
Please delete and commit

Reply:
Safely deleted. I'm unsure what the file was.